packages/devextreme/js/__internal/core/ai_integration/templates/index.ts

```diff
@@ -31,4 +31,8 @@ export const templates: PromptTemplates = {
     system: 'You are a helpful assistant that helps to fill fields based on the text provided. You will get a text and a list of fields that should be filled using info from the text. It can include the name of field, suitable format, optionally some additional instruction about what it should include. You need to return data for all the fields in the following format without any preamble, introduction, or explanatory text: {fieldName}:::{fieldValue};;;{fieldName}:::{fieldValue} and so on, where {fieldName} - is a variable for a field name and {fieldValue} - is a variable for a string to fill. If there is no info to fill, field value should be empty (like Name:::;;;)- do not use placeholders like (empty), N/A, null, or similar. Only fill in date fields if a complete date is explicitly present. If the date is missing or incomplete, leave the field empty.',
     user: 'Text: {{text}}. Fields: {{fields}}.',
   },
+  generateColumn: {
```

Comment:
[thoughts]
This API may become public, so I would ask TechWriters to help with the name. I would narrow down the meaning of this name to `generateDataGridColumn`.

Reply:
I agree, adding a bit of specificity is beneficial. Let's rename this template to `generateGridColumn`. This adds specificity while keeping the name relatively short and leaving the door open to using this template in the TreeList (if we ever decide to add AI columns to the TreeList).
```suggestion
  generateGridColumn: {
```